avm/ptn/azd/insights-dashboard/main.bicep

```diff
@@ -0,0 +1,109 @@
+metadata name = 'Application Insights Components'
```

Comment:
Softblocker until MAR file is updated
cc: @matebarabas

Reply:
FYI, the required record is currently in PR, along with all the records of the other AZD ptn modules. I'll let you know when it's merged.